x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/setup_page/index.tsx

```diff
@@ -87,23 +87,23 @@ export const SetupPage: React.FunctionComponent<{
               <h2>
                 <FormattedMessage
                   id="xpack.ingestManager.setupPage.enableTitle"
-                  defaultMessage="Enable Fleet"
+                  defaultMessage="Enable central management for Elastic Agents"
                 />
               </h2>
             </EuiTitle>
             <EuiSpacer size="xl" />
             <EuiText color="subdued">
               <FormattedMessage
                 id="xpack.ingestManager.setupPage.enableText"
-                defaultMessage="Fleet requires an Elastic user who can create API keys and write to logs-* and metrics-*."
+                defaultMessage="Central management for Elastic Agents requires an Elastic user who can create API keys and write to logs-* and metrics-*."
```

Comment:
You don't need to repeat "Elastic Agents" here since it's established in the title.